change-notes/1.20/analysis-python.md

```diff
@@ -0,0 +1,29 @@
+# Improvements to Python analysis
+
+
+ ## General improvements
+
+ > Changes that affect alerts in many files or from many queries
+> For example, changes to file classification
+ ## New queries
+
+ | **Query**                   | **Tags**  | **Purpose**                                                        |
+|-----------------------------|-----------|--------------------------------------------------------------------|
+| Default version of SSL/TLS may be insecure (`py/insecure-default-protocol`) | security, external/cwe/cwe-327 | Results are shown on LGTM by default. |
+| Use of insecure SSL/TLS version (`py/insecure-protocol`) | security, external/cwe/cwe-327 | Results are shown on LGTM by default. |
+
+ ## Changes to existing queries
+
+ All taint-tracking queries now support visualization of paths in QL for Eclipse.
+Most security alerts are now visible on LGTM by default.
```

Comment:
I think that the two lines above may be leftover from the 1.19 release notes. Please can you check?

Reply:
You're absolutely right. Thanks for pointing this out.